.azure-pipelines/client.yml

```diff
@@ -44,8 +41,6 @@ jobs:
           artifactName: packages
 
   - job: 'Analyze'
```

Comment:
While we're here. Maybe update the installation of `doc-warden` to install specific version?

`pip install doc-warden==0.3.0`